source/extensions/config_subscription/grpc/xds_mux/grpc_mux_impl.h

```diff
@@ -248,6 +260,11 @@ class GrpcMuxDelta : public GrpcMuxImpl<DeltaSubscriptionState, DeltaSubscriptio
   // GrpcStreamCallbacks
   void requestOnDemandUpdate(const std::string& type_url,
                              const absl::flat_hash_set<std::string>& for_update) override;
+
+private:
+  std::string methodName() const override {
+    return "envoy.service.discovery.v3.AggregatedDiscoveryService.DeltaAggregatedResources";
```

Comment:
could this be factored out as constant and method will return string_view instead?

Reply:
returning an absl::string_view.
Regarding the constant, I have a vague recollection from my early C days, that the compiler puts the fixed strings from the code in the static data section of the process. It should then point to that contents in memory in the rest of the code (this was true for `const char*`, and I assume it will be the same for string_view).